### PR TITLE
Gateway rate limiting landing page

### PR DESCRIPTION
**Pull request by mheap**
https://deploy-preview-46--kongdeveloper.netlify.app/gateway/rate-limiting/

### CODE REVIEW

File: app/_landing_pages/gateway/rate-limiting.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+                    Rate limits can be applied to a [service](/gateway/entities/service/) or individual [routes](/gateway/entities/route/) to protect the upstream API, or to a [consumer](/gateway/entities/consumer/) to restrict how many times a user can call the API in the specified period.
+
+                    In addition to the OSS Rate Limiting plugin, Kong offer an Enterprise rate limiting plugin with additional capabilities.
+  - row:
+      - column:
+          - type: structured_text
+            config:
+              header:
+                text: "Open Source Features"
```
Comment (cloudjumpercat) [inline app/_landing_pages/gateway/rate-limiting.yaml:31]: I think this formatting works, but I think for a feature comparison, a feature support table might work better to get info at a glance.

Comment (fabianrbz) [reply app/_landing_pages/gateway/rate-limiting.yaml:31]: I'd say let's ship this, we can ask the design team for feedback

Comment (mheap) [reply app/_landing_pages/gateway/rate-limiting.yaml:31]: This definitely isn't the end result. The content is indicative, but it needs a ton of design work / layout fixes

**Review by cloudjumpercat — APPROVED** [review]
LGTM! And I'd like to use these changes in my PR

**Pull request merged**